package.json

```diff
@@ -143,4 +146,4 @@
     "loader-utils": "2.0.3"
   },
   "homepage": "https://welcome-ui.com"
-}
+}
```

Comment:
**depcheck** seems to delete the last line of package.json files, maybe we need to run **prettier** after?

Reply:
Je crois que c'est mon VSCode en fait.

Reply:
can you fix this? 🤔

Reply:
Ça y est. 🤓